gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStoragePath.java

```diff
@@ -273,9 +277,9 @@ public int compareTo(Path other) {
   public boolean equals(@Nullable Object other) {
     return this == other
         || other instanceof CloudStoragePath
-        && Objects.equals(bucket(), ((CloudStoragePath) other).bucket())
-        && Objects.equals(toRealPathInternal(false),
-            ((CloudStoragePath) other).toRealPathInternal(false));
+            && Objects.equals(bucket(), ((CloudStoragePath) other).bucket())
```

Comment:
I mentioned it in the prior PR but I don't remember an answer (could be my fault).
FindBugs complains that setting `other` as `@Nullable` is wrong as it is not compatible with the method it overrides.
Clearly, it is a harmless message but I would still like to raise (I am actually not sure what is the recommendation for it).